src/js/libgeo/GeoOps.js

```diff
@@ -2426,6 +2441,9 @@ geoOps.TrReflectionP.updatePosition = function(el) {
 geoOps.TrReflectionL = {};
 geoOps.TrReflectionL.kind = "Tr";
 geoOps.TrReflectionL.signature = ["L"];
+geoOps.TrReflectionL.initialize = function(el) {
+    el.isEuclidean = -1;
```

Comment:
A property which is called "is*" i would have expected something of type bool. Not a show-stopper.